src/components/SignInSteps/SignInStepsStepper.tsx

```diff
@@ -0,0 +1,41 @@
+import { useRouterQuery } from '@/hooks'
+import React from 'react'
+import { useNavigate } from 'react-router'
+import { Multistepper } from '../Dialogs'
+import AccountStep from './AccountStep'
+import ExtensionStep from './ExtensionStep'
+import TermsStep from './TermsStep'
+
+type SignInStepsStepperProps = {
+  path: (query: { step: string }) => string
```

Comment:
Does this component need this path? I think it could just use the route at which it is displayed and only change the query param?